server/app.es6

```diff
@@ -76,6 +99,8 @@ app.get('/uptime', function(req, res) {
   res.send('Uptime: ' + process.uptime() + ' seconds');
 });
 
+app.use(loginGov.router);
```

Comment:
Again, without this being a fully implemented feature, I'd rather have it not be part of the sprint prs.